blocks/secondary-navbar/secondary-navbar.css

```diff
@@ -1,125 +1,297 @@
+.tabs-pane-js {
+  max-width: 1440px;
+  position: relative;
+  left: 70px;
+  box-sizing: border-box;
+}
+
+.tab-pane {
   display: none;
 }
-.secondary-nav-cta-wrapper {
-  flex-shrink: 0;
-  margin-left: auto;
+
+.tab-pane.active {
+  display: block;
 }
-/* Base nav item */
-.secondary-nav-item {
+
+/* Navigation list */
+.top-nav-left {
+  display: flex;
+  gap: 48px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  align-items: center;
+  justify-content: flex-start;
+  box-sizing: border-box;
+}
+
+.top-nav-left > li {
+  position: relative;
+  padding: 14px 0;
+}
+
+.top-nav-left > li > a {
+  color: #333;
+  font-size: 13px;
   font-weight: 400;
-  color: #25243B !important;
-  font-size: 17px;
-  line-height: 20px;
-  letter-spacing: -0.5px;
-  padding: 25px 0px 20px;
+  line-height: 18px;
   text-decoration: none;
-  position: relative;
-  display: inline-block;
+  cursor: pointer;
   white-space: nowrap;
-  transition: font-weight 0.2s ease;
+  transition: color 0.2s ease;
+}
+
+.top-nav-left > li > a:hover {
+  color: #9d1d27;
+}
+
+/* Dropdown Styles */
+.drop-down {
+  cursor: pointer;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  top: 100%;
+  left: 50%;
+
+  /* transform: translateX(-50%); */
+  margin-top: 8px;
+  border-radius: 0 8px 8px;
+  box-shadow: 0 4px 16px rgb(0 0 0 / 12%);
+  background: #fff;
+  border: 1px solid #e0e0e0;
+  min-width: 720px;
+  max-height: 320px;
+  overflow-y: auto;
+  z-index: 1000;
+}
+
+.drop-down:hover > .dropdown-content {
+  display: block;
 }
-.secondary-nav-item:hover {
+
+/* Header Box */
+.hd-bx {
+  padding: 18px 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #f0f0f0;
+  background: #fafafa;
+}
+
+.hd-bx-h4 {
+  margin: 0;
+  font-size: 15px;
+  font-weight: 600;
+  color: #333;
+}
+
+.hd-bx .link {
+  color: #9d1d27;
+  font-size: 13px;
   text-decoration: none;
   font-weight: 500;
 }
-.secondary-nav-item:hover::after {
-  content: "";
-  position: absolute;
-  bottom: 0;
-  left: 0;
-  height: 3px;
-  width: 100%;
-  background-color: #9D1D27;
-  transition: all 0.3s ease;
-}
-/* CTA Button */
-.secondary-nav-cta {
-  display: inline-block;
-    color: rgb(255, 255, 255) !important;
-    text-align: center;
-    font-size: 16px;
-    font-weight: 500;
-    letter-spacing: -0.5px;
-    line-height: 22px;
-    padding: 12px 30px;
-    border-radius: 25px;
-    background: rgb(157, 29, 39);
-    white-space: nowrap;
-}
-.secondary-nav-cta:hover {
-  background-color: #A01313;
-  text-decoration: none !important;
-  border: none;
-}
-.secondary-nav-item.secondary-nav-cta:hover::after {
-  display: none;
+
+.hd-bx .link:hover {
+  text-decoration: underline;
+}
+
+/* Menu Card List */
+.menu-cardList {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 20px 24px;
+}
+
+.menu-cardList.MT15 {
+  margin-top: 0;
+}
+
+/* Gradient Cards */
+.grdiantCard {
```

Comment:
Is there a reason you're not using '.gradient' for the class name here? I'll approve and merge but maybe when we go to update the hexadecimal colors to css variable names we can fix the ESL spelling too.